core/snippets/snippets/methodDeclaration.snippet

```diff
@@ -0,0 +1,13 @@
+name: methodDeclaration
```

Comment:
we don't support scoping by scope type right? If so, we could automatically do this for functions when we're in a class, like how we do with the default Cursorless function snippet https://github.com/cursorless-dev/cursorless/blob/d6443ffc0e46e9fbc7eb262645c7cf17a5ea8c3b/cursorless-snippets/functionDeclaration.cursorless-snippets#L24-L49

Reply:
Correct no support for that at the moment

Reply:
Looks like we don't even support that on the Cursorless snippet command api surface yet